src/commands/project/deploy/pipeline/validate.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Messages } from '@salesforce/core';
+import { PromoteOptions, PromotePipelineResult } from '../../../../common';
+import DeployPipeline from './start';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-devops-center', 'project.deploy.pipeline.validate');
+
+/**
+ * Contains the logic to execute the sf project deploy pipeline start command.
+ */
+export default class validateDeployPipeline extends DeployPipeline {
```

Comment:
```suggestion
export default class ValidateDeployPipeline extends DeployPipeline {
```

Let's stay consistent